src/middleware/sync_on_becoming_visible.js

```diff
@@ -0,0 +1,27 @@
+import { sync } from '../actions/org';
+import { debounce } from 'lodash';
+
+function dispatchSync(store) {
```

Comment:
Hey @munen, sorry to nit, but I've had a pretty strong preference for arrow functions in org-web instead of old-style functions. So something like this: 

```
const dispatchSync = store => store.dispatch(sync({ shouldSuppressMessages: true }));
```

Also, have you installed Prettier? I noticed an indentation issue above that I think Prettier would've picked up on

Reply:
@DanielDe Nitting is good. Those are my first of hopefully many contributions. It's good if we get a consistent feel on how the codebase should evolve. Just keep commenting^^

I'll gladly change it to a arrow function - and have a suggestion: What do you think of having an extensive eslintrc file with suggestions like these? This particular one isn't in the standard eslint functionality, though (only for callbacks).

Regarding prettier, of couse I have it (because I use it myself [see my `autoformat` Emacs integration: https://github.com/munen/emacs.d/#auto-formatting] and because I read your contribution guidelines^^). Running it on the file yields no changes to me.

Reply:
Done.

Reply:
The indentation issue was in a different file, but I'll look into it. Maybe I was just wrong about how Prettier would format that! 

I'm not a huge fan of linters myself, though I'm open to it if it seems necessary in the future.